addons/web/static/src/js/fields/basic_fields.js

```diff
@@ -1363,40 +1365,31 @@ var CopyClipboard = {
             });
         });
     },
-};
-
-var TextCopyClipboard = FieldText.extend(CopyClipboard, {
-
-    //--------------------------------------------------------------------------
-    // Private
-    //--------------------------------------------------------------------------
-
     /**
      * @override
      */
-    _render: function() {
+    _render: function () {
         this._super.apply(this, arguments);
         this.$el.addClass('o_field_copy');
-        this.$el.append($(qweb.render('CopyClipboardText')));
-        this._initClipboard();
-    }
-});
-
-var CharCopyClipboard = FieldChar.extend(CopyClipboard, {
-
-    //--------------------------------------------------------------------------
-    // Private
-    //--------------------------------------------------------------------------
-
+    },
     /**
      * @override
      */
-    _render: function() {
+    _renderReadonly: function () {
```

Comment:
_render and _renderReadonly return a promise, so we should return the result of calls to _super